### PR TITLE
update notebook guidelines

### PR DESCRIPTION
**Pull request by pablo-gar**
- Corrects mentions of old feature name "Cell Census"
- Adds symlink information for python notebooks

### CODE REVIEW

File: docs/census_notebook_guidelines.md [hunk 1]
```diff
@@ -2,9 +2,10 @@
 
 API demonstration code that is part of the documentation should be deposited here: 
 
-- Python notebooks [`cell-census/api/python/notebooks`](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks)
-- R vignettes [`cell-census/api/r/CellCensus/vignettes`](https://github.com/chanzuckerberg/cell-census/tree/main/api/r/CellCensus/vignettes)
+- Python notebooks [`cellxgene-census/api/python/notebooks`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/api/python/notebooks)
+- R vignettes [`cellxgene-census/api/r/CellCensus/vignettes`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/api/r/cellxgene.census/vignettes)
 
+If the Python notebooks have to be included in the doc-site, then a symlink has to be created in the corresponding folder of [ `cellxgene-census/docs/notebooks/`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/docs/notebooks).
```
Comment (bkmartinjr) [inline docs/census_notebook_guidelines.md:8]: Suggested edit: 
> To include Python notebooks in the doc site, create a symlink to the notebook in the [ `cellxgene-census/docs/notebooks/`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/docs/notebooks) folder.

File: docs/census_notebook_guidelines.md [hunk 2]
```diff
@@ -13,7 +14,7 @@ These assets are user-facing and are automatically rendered to the doc-sites, th
 ### Title
 
 * It must use the highest-level markdown header `#`.
-* Unless needed, it should not contain "Cell Census".
+* Unless needed, it should not contain any direction mentions of "Census".
```
Comment (bkmartinjr) [inline docs/census_notebook_guidelines.md:17]: Is there an alternative, or is the request to attempt no use of a "noun" to reference the collection?

**Review by bkmartinjr — APPROVED** [review]
Couple of minor questions/suggestions, but no major concerns.

File: docs/census_notebook_guidelines.md [hunk 3]
```diff
@@ -31,19 +32,19 @@ Examples:
...
 * It must provide a one paragraph summary of the notebook's goals.
-* It must not contain an explanation of the Cell Census.
+* It must not contain an explanation of the Census.
 
 Examples: 
 
 :white_check_mark:
 
-> This notebook provides a demonstration for integrating two Cell Census datasets using scvi-tools. The goal is not to provide an exhaustive guide on proper integration, but to showcase what information in the Cell Census can inform data integration.
+> This notebook provides a demonstration for integrating two Census datasets using scvi-tools. The goal is not to provide an exhaustive guide on proper integration, but to showcase what information in the Cell Census can inform data integration.
```
Comment (bkmartinjr) [inline docs/census_notebook_guidelines.md:41]: confusing as it mentions "Census" - how does this tie into the "no direction mention" guideline?

File: docs/census_notebook_guidelines.md (hunk 2 above)
Comment (ebezzi) [reply docs/census_notebook_guidelines.md:17]:
```suggestion
* Unless needed, it should not contain any direct mentions of "Census".
```
Is that what you meant?

File: docs/census_notebook_guidelines.md (hunk 3 above)
Comment (atolopko-czi) [reply docs/census_notebook_guidelines.md:41]: isn't that rule just for the title?

Comment (atolopko-czi) [inline docs/census_notebook_guidelines.md:41]:
```suggestion
> This notebook provides a demonstration for integrating two Census datasets using scvi-tools. The goal is not to provide an exhaustive guide on proper integration, but to showcase what information in the Census can inform data integration.
```

File: docs/census_notebook_guidelines.md [hunk 4]
```diff
@@ -2,9 +2,10 @@
...
 API demonstration code that is part of the documentation should be deposited here: 
 
-- Python notebooks [`cell-census/api/python/notebooks`](https://github.com/chanzuckerberg/cell-census/tree/main/api/python/notebooks)
-- R vignettes [`cell-census/api/r/CellCensus/vignettes`](https://github.com/chanzuckerberg/cell-census/tree/main/api/r/CellCensus/vignettes)
+- Python notebooks [`cellxgene-census/api/python/notebooks`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/api/python/notebooks)
+- R vignettes [`cellxgene-census/api/r/CellCensus/vignettes`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/api/r/cellxgene.census/vignettes)
 
+If the Python notebooks have to be included in the doc-site, then a symlink has to be created in the corresponding folder of [ `cellxgene-census/docs/notebooks/`](https://github.com/chanzuckerberg/cellxgene-census/tree/main/docs/notebooks).
 
 These assets are user-facing and are automatically rendered to the doc-sites, they are one of the primary means by which users get onboarded to the product. Thus the following guidelines need to be followed to ensure readability and a consistent experience.
```
Comment (atolopko-czi) [inline docs/census_notebook_guidelines.md:10]:
```suggestion
These assets are user-facing and are automatically rendered to the doc-site and are one of the primary means by which users get onboarded to the product. Thus the following guidelines need to be followed to ensure readability and a consistent experience.
```

File: docs/census_notebook_guidelines.md (hunk 1 above)
Comment (pablo-gar) [inline docs/census_notebook_guidelines.md:8]:
```suggestion
To include Python notebooks in the doc site, create a symlink to the notebook in [cellxgene-census/docs/notebooks/](https://github.com/chanzuckerberg/cellxgene-census/tree/main/docs/notebooks) folder.
```

**Pull request merged**